packages/passport/sdk/src/starkEx/workflows/transfer.ts

```diff
@@ -145,6 +141,7 @@ export async function batchNftTransfer({
       });
 
       const requests = await Promise.all(
+        // @ts-ignore - TS is not able to infer the type of the response
```

Comment:
Why it becomes a problem.
what's the difference between `core-sdk` and `x-client`'s **getSignableTransfer** return?

Reply:
Seems like my VSCode is not complaining about this anymore...